pymc/util.py

```diff
@@ -28,6 +28,12 @@
 from cachetools import LRUCache, cachedmethod
 
 
+def as_list(obj):
+    if not isinstance(obj, Sequence):
+        obj = [obj]
+    return list(obj)
+
+
```

Comment:
This already exists here: https://github.com/pymc-devs/pymc/blob/12f4cd73bae95c42a28c2f07f6aab60071fd2ca1/pymc/aesaraf.py#L546-L550

Maybe move it to `util.py`?